### PR TITLE
Update Read-Me

### PR DESCRIPTION
**Pull request by ayushnawal**
Previously there were no specific instruction on how to clone the repository.

Now adding a guideline for cloning the repository through git ssh.

Fixes https://github.com/internetarchive/openlibrary/issues/1834

> **Description**: What does this PR achieve? [feature|hotfix|refactor]

Closes #

> **Technical**: What should be noted about the implementation?

> **Testing**: Steps for reviewer to reproduce / verify this PR fixes the problem?

> **Evidence**: If this PR touches UI, please post evidence (screenshot) of it behaving correctly:

### CODE REVIEW

**Comment by tabshaikh** [comment]
@ayushnawal i did a normal git and it works fine, were you working with windows on this

**Comment by ayushnawal** [comment]
@tabshaikh  maybe the setup is working fine for you because older docker images are used by docker-compose up in your machine . as they are cloned and built properly , therefore it's working fine i guess

**Comment by tabshaikh** [comment]
Okay I will review this pr as I have the bandwidth for a fresh installation:)

**Comment by ayushnawal** [comment]
@tabshaikh any update sir ?

File: Readme.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ Open Library is an effort started in 2006 to create "one web page for every book
 
 ## Installation
 
+Thanks for your interest in improving the openlibrary ! .
+Very first step is to [fork and clone](https://help.github.com/articles/fork-a-repo/).
+We recommend you to clone the OL github repository using [Git SSH](https://help.github.com/articles/connecting-to-github-with-ssh/) for proper functioning of our git submodules .
```
Comment (mekarpeles) [inline Readme.md]: Is this right? Or do we want to be using `http`?

File: Readme.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ Open Library is an effort started in 2006 to create "one web page for every book
 
 ## Installation
 
+Thanks for your interest in improving the openlibrary ! .
```
Comment (mekarpeles) [inline Readme.md]: let's remove this line please

File: Readme.md [hunk 3]
```diff
@@ -29,6 +29,10 @@ Open Library is an effort started in 2006 to create "one web page for every book
...
+Very first step is to [fork and clone](https://help.github.com/articles/fork-a-repo/).
```
Comment (mekarpeles) [inline Readme.md]: this should be under the **Docker** section

File: Readme.md [hunk 4]
```diff
@@ -54,7 +58,7 @@ You can also find more information regarding Developer Documentation for Open Li
 
 ### The Backend
 
-OpenLibrary is developed on top of the Infogami wiki system, which is itself built on top of the web.py Python web framework and the Infobase database framework. 
+OpenLibrary is developed on top of the Infogami wiki system, which is itself built on top of the web.py Python web framework and the Infobase database framework.
```
Comment (mekarpeles) [inline Readme.md]: 👍

**Comment by mekarpeles** [comment]
Closing as https://github.com/internetarchive/openlibrary/pull/1856 was merged -- thank you for initiating this PR @ayushnawal!

**Pull request closed without merging**